docs/resources/synthetic_monitoring_check_alerts.md

```diff
@@ -0,0 +1,65 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "grafana_synthetic_monitoring_check_alerts Resource - terraform-provider-grafana"
+subcategory: "Synthetic Monitoring"
+description: |-
+  Manages alerts for a check in Grafana Synthetic Monitoring.
+  These alerts will trigger when your check fails from the specified probes.
+---
+
+# grafana_synthetic_monitoring_check_alerts
+
+Manages alerts for a check in Grafana Synthetic Monitoring.
+
+* [Official documentation](https://grafana.com/docs/grafana-cloud/testing/synthetic-monitoring/configure-alerts/configure-per-check-alerts/)
+* [API documentation](https://github.com/grafana/synthetic-monitoring-api-go-client/blob/main/docs/API.md#alerts)
```

Comment:
I made a [comment](https://github.com/grafana/synthetic-monitoring-api-go-client/pull/272#pullrequestreview-2775499104) on this in the PR for the API client that this docs currently don't exist.